src/windows/views/blender_listview.py

```diff
@@ -735,6 +735,13 @@ def __init__(self, blend_file_path, target_script, preview_frame=0):
         self.version = None
         self.process = None
         self.canceled = False
+        self.env = dict(os.environ)
+        if sys.platform == "linux" and os.path.exists('/lib/x86_64-linux-gnu/'):
+            # If on Linux, verify we have the following LD library path defined.
+            # This is needed for Blender to use the system libraries instead
+            # of our AppImage libraries (i.e. our libtiff.so.5 is missing symbols,
+            # compared to libtiff.so.5 on newer distros for some reason).
+            self.env['LD_LIBRARY_PATH'] = '/lib/x86_64-linux-gnu/'
```

Comment:
Is there any significance to this `x86_64` part? I'm wondering if that's correct on AMD systems.

Reply:
Great question. We might need to handle things differently for different CPUs, just depending on how distros handle this.